### ISSUE TITLE
[Request] Add information about static analysis

### ISSUE THREAD

**Issue by khuei** [purduesigbots/BLRS-Wiki#16]
Static analysis can be useful for providing/detecting valuable information such as bugs, memory leaks, logic errors, cyclometric complexity, and more. This is opinionated so feel free to close this issue.

**Comment by WillXuCodes** [comment]
Agreed, we could also create a debugging subsection in the software section for topics like this.
